bundles/org.openhab.binding.openweathermap/src/main/java/org/openhab/binding/openweathermap/internal/handler/OpenWeatherMapOneCallHandler.java

```diff
@@ -681,7 +681,7 @@ private void updateAlertsChannel(ChannelUID channelUID, int count) {
         List<Alert> alerts = localWeatherData != null ? localWeatherData.alerts : null;
         State state = UnDefType.UNDEF;
         if (alerts != null && alerts.size() > count) {
-            Alert alert = alerts.get(count - 1);
+            Alert alert = alerts.get(count);
```

Comment:
Please rename parameter `count` into `index` and update the javadoc description to mention that the expected value is from 0.

Reply:
updated the javadoc (also for the other methods)